cea/utilities/rename_building.py

```diff
@@ -0,0 +1,69 @@
+"""
+A simple CEA script that renames a building in the input files - NOTE: you'll have to re-run the simulation and
+analysis scripts to get the changes as only the files defined in ``inputs.yml`` (the files you see in the CEA Dashboard
+input editor) are changed.
+
+This is the script behind ``cea rename-building --from <building> --to <building>``
```

Comment:
Is this supposed to be ``cea rename-building --old <building> --new <building>`` ?

Reply:
you're right! fixed!